ballerina/client.bal

```diff
@@ -13,3 +16,2323 @@
+        string resourcePath = string `/file_requests/delete`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # delete_all_closed
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - delete_all_closed response 
+    resource isolated function post file_requests/delete_all_closed(DeleteAllClosedHeaders headers = {}) returns inline_response_200_9|error {
+        string resourcePath = string `/file_requests/delete_all_closed`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # get
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - get response 
+    resource isolated function post file_requests/get(file_requests_get_body payload, GetHeaders headers = {}) returns inline_response_200_8|error {
+        string resourcePath = string `/file_requests/get`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    resource isolated function post file_requests/list/'continue(list_continue_body payload, ListContinueHeaders headers = {}) returns inline_response_200_10|error {
+        string resourcePath = string `/file_requests/list/continue`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # list
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - list response 
+    resource isolated function post file_requests/list_v2(file_requests_list_v2_body payload, ListHeaders headers = {}) returns inline_response_200_10|error {
+        string resourcePath = string `/file_requests/list_v2`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # update
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - update response 
+    resource isolated function post file_requests/update(file_requests_update_body payload, UpdateHeaders headers = {}) returns inline_response_200_8|error {
```

Comment:
This naming can be addressed to align with Ballerina Naming convention via this solution https://github.com/ballerina-platform/module-ballerinax-dropbox/pull/3#issuecomment-2440562817